dsl_parser/elements/operation.py

```diff
@@ -295,7 +295,8 @@ def _resource_exists(location_bases, resource_name):
         # pointing to the same installed one.
         script_plugins = [plugin for plugin in plugins
                           if constants.SCRIPT_PLUGIN_NAME in plugin]
-        if not script_plugins:
+        script_plugin = script_plugins and script_plugins[0]
```

Comment:
kinda dont like the shortcircuiting `and` here, since we use the value assuming it is the name not a bool; what if `script_plugins[0]` is false :/ (an empty string)
maybe rewrite as an if?